integration_test/third_party_apps_test/applications/dcgm/debian_ubuntu/install

```diff
@@ -38,7 +38,7 @@ case $DEVICE_CODE in
             # cuda-12-6 is the latest version that supports Debian 11
             sudo apt -y install cuda-12-6
         else
-            sudo apt -y install cuda
+            sudo apt -y install nvidia-driver-580 cuda-12-9
```

Comment:
CUDA 12.x requires driver <580: https://docs.nvidia.com/deploy/cuda-compatibility/minor-version-compatibility.html

This page suggests driver 575: https://developer.nvidia.com/cuda-12-9-0-download-archive

Although there's a versionless package you can try too: `sudo apt-get install -y nvidia-open` or `sudo apt-get install -y cuda-drivers`

Reply:
It seems like `sudo apt -y install cuda` would install a meta package that includes an Nvidia driver. However, `sudo apt -y install cuda-12-9` seems to install a smaller bundle and doesn't include an Nvidia driver. This is because I got the `nvidia-smi` command not found error, and that's why i am looking to install nvidia driver explicitly. 

I will try the cmd you shared. Thanks